bin/v6_server.ml

```diff
@@ -88,4 +88,4 @@ let apply_edition dbg edition params =
 
 let get_editions dbg = editions
 
-let get_version dbg = Version.version
+let get_version dbg = (Version.version, "final")
```

Comment:
This works like a key-value list now, This should give you flexibility on the attributes you want to pass around::
```suggestion
let get_version dbg = [("version", Version.version)]
```
or maybe:
```suggestion
let get_version dbg = [
    ("version", Version.version)
  ; ("is_final", "true")
]
```

As I said, it's up to xcp-ng on what it wants to offer here